IsogeomGenerator/generate_isogeom.py

```diff
@@ -0,0 +1,332 @@
+
+    # only have option to generate levels if not moab mode
+    if not moab:
+        level_group.add_argument('-gl', '--generatelevels',
+                                 action='store',
+                                 nargs=1,
+                                 choices=['ratio', 'log', 'lin'],
+                                 default=None,
+                                 metavar='ratio/log/lin',
+                                 type=str,
+                                 help='Specifies the mode for generating ' +
+                                 'level values to be used for the ' +
+                                 'isosurfaces. '
+                                 'If used, values for the minimum and ' +
+                                 'maximum levels (-lex) and the ratio or ' +
+                                 'number of levels (-N) are also required. ' +
+                                 'Options are: ' +
+                                 '(1) ratio: N is the ratio between levels ' +
+                                 'ranging from the min value upto, but not ' +
+                                 'exceeding, the max value. ' +
+                                 '(2) log: N is the number of levels to be ' +
+                                 'evenly spaced logarithmically between the ' +
+                                 'min and max values. ' +
+                                 '(3) lin: N is the number of levels to be ' +
+                                 'evenly spaced linearly between the min ' +
+                                 'and max values.'
+                                 )
+        parser.add_argument('-lx', '--levelextrema',
+                            action='store',
+                            nargs=2,
+                            required=False,
+                            default=None,
+                            metavar='MIN_VAL MAX_VAL',
+                            dest='extN',
+                            type=float,
+                            help='float, minimum and maximum values to use ' +
+                            'for generating the set of level values to be ' +
+                            'used for the isosurfaces.'
+                            )
+        parser.add_argument('-N', '--numlevels',
+                            action='store',
+                            nargs=1,
+                            required=False,
+                            default=None,
+                            metavar='N',
+                            dest='N',
+                            type=float,
+                            help='If generating levels (-gl), it is either ' +
+                            'the ratio between adjacent level values (ratio ' +
+                            'mode), or the number of levels to generate ' +
+                            '(log or lin mode).'
+                            )
+
+
+def set_visit_only_options(parser):
+    """set options specific to the Visit step.
+    """
```

Comment:
would add the list of arguments that can be parsed here.

Reply:
I think this is resolved with the addition to the README.